{{cookiecutter.repo_name}}/README.rst

```diff
@@ -9,53 +13,50 @@
+Source Code
+===========
 
-.. seealso::
+The {{ cookiecutter.project_name }} source code can be found in the `Github repo`_ or in the `Distributions`_ folder within the Anmut Hub - Software Products SharePoint.
 
-	For more detailed installation instructions, see :ref:`installation`
+For more on installing from source, see :ref:`installation`.
 
-.. _SSH set up on GitHub: https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
+.. _Github repo: https://github.com/anmut-consulting/{{cookiecutter.repo_name}}
+.. _Distributions:
 
-To activate the environment (assuming you are using one),
-again from the root directory of the repo::
+Documentation
+=============
 
-    $ source env/bin/activate
+To read the documentation, please download (or sync) the relevant version folder from the `Documentation`_ folder in the Anmut Hub - Software Products SharePoint.
 
-And to deactivate the environment::
+If you are working from source, you can generate the docs using the following command in terminal::
 
-    $ deactivate
+    $ pipenv run make_docs
 
-Using pipenv
-------------
-if you're using pipenv to manage your environment, you need to add a #egg=display-name to the end of the install instruction.  i.e::
+Once generated, information can be found in:
 
-    $ pipenv install git+ssh://git@github.com/anmut-consulting/{{ cookiecutter.repo_name }}.git@latest#egg={{ cookiecutter.repo_name }}
+- **User Guide**: To read the usage documentation, please see :ref:`usage`.
 
-To activate the environment, the command is::
+.. _Documentation:
 
-    $ pipenv shell
+Contributions
+=============
 
-You should see ({{ cookiecutter.repo_name }}) at the start of the prompt now.  This indicates you are in the activated environment.  If you don't see it - something has gone wrong.
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
 
-To deactivate the environment again, simply run::
-
-    $ exit
+A detailed overview on how to contribute can be found in the :ref:`contributing` guide.
 
 Credits
 =======
 
-This package was created with `Cookiecutter <https://cookiecutter.readthedocs.io>`_ and the `anmut-consulting/pipenv-cookiecutter <https://github.com/anmut-consulting/pipenv-cookiecutter>`_ project template.
+This package was created with `Cookiecutter`_ and the `anmut-consulting/pipenv-cookiecutter`_ project template.
+
+.. _Cookiecutter: https://cookiecutter.readthedocs.io
+.. _anmut-consulting/pipenv-cookiecutter: https://github.com/anmut-consulting/pipenv-cookiecutter
 
-.. |GitHub Test| image:: https://github.com/anmut-consulting/{{cookiecutter.repo_name}}/workflows/Test/badge.svg
-   :target: https://github.com/anmut-consulting/{{cookiecutter.repo_name}}/actions
+.. |GitHub Test| image:: https://github.com/anmut-consulting/vdva/workflows/Test/badge.svg
+   :target: https://github.com/anmut-consulting/vdva/actions
```

Comment:
Whoops...these `vdva` parts should be replaced with `{{cookiecutter.repo_name}}` again.

Reply:
great catch, thanks!